packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -1246,3 +1246,76 @@ export const LinkExample = () => {
     </Link>
   );
 };
+
+export const ToolbarExample = (props) => {
+  return (
+    <Toolbar {...props}>
+      <ToggleButton className={classNames(styles, 'toggleButtonExample')}><strong>B</strong></ToggleButton>
+      <ToggleButton className={classNames(styles, 'toggleButtonExample')}><div style={{textDecoration: 'underline'}}>U</div></ToggleButton>
+      <ToggleButton className={classNames(styles, 'toggleButtonExample')}><i>I</i></ToggleButton>
+      {/* radio does not work in toolbar yet, selection doesn't follow focus
```

Comment:
according to the pattern, these should all work, currently they don't due to various aspects of our focus management